src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -105,6 +116,26 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
                 $numFundingsOvermatched++;
                 $overmatchAmount = bcsub($fundingWithdrawalTotal, $campaignFundingAllocated, 2);
                 $output->writeln("Funding {$funding->getId()} is over-matched by $overmatchAmount. $details");
+
+                if ($mode === '2024-06-05-patch' && in_array($funding->getId(), $problemIds, true)) {
+                    // Over match expected to be £300 but safest to drive it from live data and also assert.
+                    \assert(bccomp($overmatchAmount, '300.00', 2) === 0);
```

Comment:
for production safety it's better to use `\MatchBot\Application\Assertion`, rather than `assert`. `assert` is intended just for dev usage and can be turned off in production.

Reply:
Done in 85837c5a972dddaf4e197fb7f5db64e4019bb923